<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UITrayMenu</class>
 <widget class="QWidget" name="UITrayMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>94</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UITrayMenu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(153, 153, 153);</string>
  </property>
  <widget class="QPushButton" name="main_pushButton">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>92</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>显示主窗口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="retrue_pushButton">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>34</y>
     <width>92</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>切换账号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_pushButton">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>67</y>
     <width>92</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>关闭程序</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
